bob-common/src/configs/cluster.rs

```diff
@@ -270,6 +270,12 @@ pub struct Cluster {
     racks: Vec<Rack>,
     #[serde(default)]
     distribution_func: DistributionFunc,
+    #[serde(default = "default_max_sequential_errors")]
+    max_sequential_errors: usize,
```

Comment:
It should be in the `node.yaml`. This setting has nothing to do with the cluster configuration